devicetypes/smartthings/dawon-zigbee-signal-interlock.src/dawon-zigbee-signal-interlock.groovy

```diff
@@ -0,0 +1,131 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "Dawon Zigbee Signal Interlock", namespace: "smartthings", author: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, genericHandler: "Zigbee") {
```

Comment:
Since this DTH is mostly based on "SmartSense Open/Closed Sensor"

I suggest to support this device in this DTH:
https://github.com/SmartThingsCommunity/SmartThingsPublic/tree/master/devicetypes/smartthings/smartsense-open-closed-sensor.src

Please add You fingerprint and all necessary tweaks in "if-statements" in the whole code and configuration methods - to avoid code duplication.

Reply:
The DTH is based on "SmartSense Open/Closed Sensor."
But it has a different UIMeta file.

'Contact sensor' -> 'Signal Interlock'
'Open' / 'Closed' -> 'Disarmed' / 'Armed'
Battery -> Do not show
Temperature -> Do not show

So it was created with a new DTH.
If we add a fingerprint to 'SmartSense Open/Closed Sensor', will the UI we want be applied?
Currently, it is displayed only in developer mode.

Reply:
I noticed You use vid: "dawon-zigbee-signal-interlock2" in Your fingerprint
Who created this UI metadata for You ?

Reply:
It may be displayed in developer mode only, because it wasn't approved yet.
Anyway, please try to add Your fingerprint with your "vid" in "SmartSense Open/Closed Sensor" DTH 
and verify (in developer mode) how does this work. 
As i mentioned before, some tweaks might be needed in the code / configuration to support Your device.

Reply:
Its UIMeta was made by a Samsung development manager.
I added a fingerprint with our vid to the "SmartSense Open/Closed Sensor" DTH, and verified that it was displayed in developer mode.
So I'll change to 'SmartSense Open/Closed Sensor' DTH.

Reply:
Should I overwrite the contents of the "Dawon Zigbee Signal Interlock" DTH with the contents of the "SmartSense Open/Closed Sensor" DTH?
Added "SmartSense Open/Closed Sensor" DTH was to IDE.
Please let me know how I should change DTH.

Reply:
Please make Your changes in SmartSense Open/Closed Sensor DTH file (so we would't have a new separate dawon-zigbee-signal-interlock.groovy file). 

1) Add Your fingerprint and VID
2) Add all configurations and other changes in a control flow regarding Your device
3) If the behavior of Your device is a bit different than for other devices - please add proper "if" statements to keep all the flows consistent. I may be useful to add a new method isDawonZigbeeSignalInterlock() {device.getDataValue("model") == "SS-B100-ZB"}.

Reply:
I added finerprintf and vid to "SmartSense Open/Closed Sensor" DTH.
And I deleted "Dawon Zigbee Signal Interlock" DTH from IDE.

Reply:
Since You added Your changes to "SmartSense Open/Closed Sensor" DTH - dawon-zigbee-signal-interlock.groovy should be deleted from this PR.

Reply:
Files changed -> dawon-zigbee-signal-interlock.groovy -> Delete file
Do I proceed with the above procedure?

Reply:
That might work. 
You may also delete that file (and folder) on Your local computer and push those changes - if You use github command line.

Reply:
I will delete the file in the above way.